coda.io.website.json

```diff
@@ -0,0 +1,31 @@
+{  
+    "providerId":"coda.io",
+    "providerName":"Coda",
+    "serviceId":"website",
+    "serviceName":"Coda Site",
+    "version": 1,
+    "syncPubKeyDomain" : "coda.io",
+    "logoUrl":"https://domainconnect.org/wp-content/uploads/2016/06/wix-300x103.png",
+    "description":"Enables a custom domain to work with Coda",
```

Comment:
Could be worthwhile to use a more specific description like, "Allows serving Coda documents from your own domain"